.github/CODEOWNERS

```diff
@@ -179,6 +179,7 @@ receiver/otlpjsonfilereceiver/                       @open-telemetry/collector-c
 receiver/postgresqlreceiver/                         @open-telemetry/collector-contrib-approvers @djaglowski
 receiver/prometheusexecreceiver/                     @open-telemetry/collector-contrib-approvers @dmitryax
 receiver/prometheusreceiver/                         @open-telemetry/collector-contrib-approvers @Aneurysm9 @dashpole
+receiver/promtailreceiver/                           @open-telemetry/collector-contrib-approvers @mar4uk
```

Comment:
Please, add @kovrus and me as well. Perhaps even @gouthamve should be included :-)

Reply:
fixed